client/ayon_core/hosts/houdini/plugins/create/create_alembic_camera.py

```diff
@@ -26,9 +27,13 @@ def create(self, subset_name, instance_data, pre_create_data):
             pre_create_data)  # type: CreatedInstance
 
         instance_node = hou.node(instance.get("instance_node"))
+
+        filepath = self.staging_dir.format(
+            product_name="`chs(\"subset\")`"  # keep dynamic link to subset
+        )
```

Comment:
I would actually do this differently. First off, I'd use similar var naming as anatomy templates, I believe there it's `{product[name]}`.

If that is passed, then we embed the hardcoded value - a studio might want that! (not sure why, but they might)

If they want to instead have the dynamic path, they can use:
```
$HIP/ayon/`chs("subset")`/`chs("subset")`.abc
```

By doing it that way - the studio can decide to use either the dynamic or hardcoded/expanded one?

Also, I can't think of any reason why anyone wanted to adjust the extension - so I'm not entirely sure whether we should expose that in the settings - but hey... whatever rows your both I suppose.

Reply:
> why anyone wanted to adjust the extension 

Some creators/rops allow artists to choose a `ext` therefore I left it with `{ext}`.
---
I'll use `product[name]` in code and an admin can make it dynamic by using `chs("subset")`